releasenotes/notes/replcmds-3a95564a5f58197b.yaml

```diff
@@ -0,0 +1,10 @@
+---
+deprecations:
+  - |
+    Begin deprecating "Agent start" command.  It is being replaced by "run".  Command
```

Comment:
I would replace `Command  will continue to function` by `The "start" command  will continue to function`, to be extra clear